tokenclient/tokenFlows.go

```diff
@@ -29,13 +29,14 @@ type Options struct {
 type RequestOptions struct {
 	// Request parameters that shall be overwritten or added to the payload
 	Params map[string]string
+	// Token Endpoint overwrites the default used /oauth2/token
+	TokenEndpoint string
 }
 
 // TokenFlows setup once per application.
 type TokenFlows struct {
 	identity env.Identity
 	Options  Options
-	tokenURI string
```

Comment:
Since `tokenURI` was private and not used anywhere I've removed it from here since setting the `tokenURI` in line 95 with the default token endpoint could confuse the user.